src/App.tsx

```diff
@@ -19,8 +22,10 @@ function App() {
 
   return (
     <ThemeProvider theme={theme}>
-      <CssBaseline />
-      <RouterProvider router={routes} />
+      <QueryClientProvider client={queryClient}>
+        <CssBaseline />
+        <RouterProvider router={routes} />
+      </QueryClientProvider>
```

Comment:
별도의 파일로 분리해서 요렇게 쓰시는 분들도 있더라구요
저는 지금도 좋습니다
```suggestion
      <ReactQueryProvider>
        <CssBaseline />
        <RouterProvider router={routes} />
      </ReactQueryProvider>
```